frontend/src/scenes/saved-insights/savedInsightsLogic.ts

```diff
@@ -17,6 +19,7 @@ interface InsightsResult {
 
 export const savedInsightsLogic = kea<savedInsightsLogicType<InsightsResult>>({
     actions: {
+        addGraph: (type: Key) => ({ type }),
```

Comment:
Is this really the type? Perhaps `menuItems` should have the correct constant key as a separate attribute and you just pass it here too